libnopegl/src/node_renderother.c

```diff
@@ -869,6 +869,7 @@ static int rendercolor_prepare(struct ngl_node *node)
     const struct pgcraft_uniform uniforms[] = {
         {.name="modelview_matrix",  .type=NGLI_TYPE_MAT4,  .stage=NGLI_PROGRAM_SHADER_VERT},
         {.name="projection_matrix", .type=NGLI_TYPE_MAT4,  .stage=NGLI_PROGRAM_SHADER_VERT},
+        {.name="aspect",            .type=NGLI_TYPE_F32,   .stage=NGLI_PROGRAM_SHADER_FRAG},
```

Comment:
nit: `renderother: provide the aspect ratio to the fragment shader`

Reply:
Done, adding a "always" (some already do)